### DISCUSSION TITLE
[Project Beta] Allow users to create views like the ones in the projects

### DISCUSSION THREAD

**Discussion by dturrin-work** [community/community#9802] [Projects and Issues]
First of all, I love the new project. Allows me to do a lot of project management activities that previously required Jira-like tools.

A comment that I received from many of the developers on my team was: _"Could I have a view of my own that I can use to track my issues in the way that works best for me?"_

My temporary solution until I can find a better way is to create a view for each developer within the project:

![image](https://user-images.githubusercontent.com/39596181/148556519-748adcc7-81e6-4bb7-ab77-b0d13ea7ce1d.png)

### Pros:

* Each developer can look at the other developer views for inspiration.
* I can see how each developer tracks their work.

### Cons:

* Views are not private.
* Views take up a lot of visual space that should be reserved for sprints, etc.
* If I want the developers to be able to customize their views, I need to give them write permissions on the project. This is not always an option.

It would be great if GitHub users could define their own account-based views based on the issues assigned to them.

I can simulate this by creating a project for every developer in the organization and then assign items to the developer's project. That, however, is a lot of extra management which most project managers do not have time for.

**Comment by ceolinwill** [comment]
This is possibly related to #6423

**Reply by Putt0** [reply]
Da uma olhadinha nesse: https://github.com/orgs/github-community/discussions/6423 ```talvez seja isso```
